pkg/zfs/cmd/v1alpha1/zpool/import/builder.go

```diff
@@ -102,7 +102,9 @@ func (p *PoolImport) WithForceImport(ForceImport bool) *PoolImport {
 
 // WithProperty method fills the Property field of PoolImport object.
 func (p *PoolImport) WithProperty(key, value string) *PoolImport {
-	p.Property = append(p.Property, fmt.Sprintf("%s=%s", key, value))
+	if len(value) != 0 {
```

Comment:
Why are we having this check? What if caller wants to unset the value?

Reply:
cc @vishnuitta ^^

Reply:
we can probably have WithPropertyUnset if required..
without this len check here, this need to be added at caller which would be at multiple places and for multiple properties